content/en/docs/reference/score-spec-reference.md

```diff
@@ -92,57 +86,32 @@ resources:
-
+| Resource type | `score-compose`                                                                                                                 | `score-humanitec`                                                                                                                                            |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `environment` | Translates to the environment variables references. For example: `${PROPERTY-NAME}`.                                            | Translates to the application values references. For example: `${values.property-name}`.                                                                     |
+| `volume`      | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification. | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification.                              |
+| `service`     | N/A                                                                                                                             | Translates to the module properties references.                                                                                                              |
+| `workload`    | N/A                                                                                                                             | Reserved resource type. Its usage may lead to compatibility issues with future releases of [score-humanitec](https://github.com/score-spec/score-humanitec). |
 
 ### Referencing resources
 
-Resources are declared in the `containers` section of the Score file and must map to the YAML structure defined in the `resources` section to properly resolve.
-
-For example, the following resource would map to the structure of the resource section.
-
-```yaml
-${resources.resource-name.property-name}
-```
-
-```yml
-resources:
-  resource-name:
-    property-name: my-property-name
-```
+Resources declared in the resources section of a Score file can be used in substitution patterns in different places
 
 {{% alert %}}
 
-> If the referenced resource or its property has not been defined, the {{< glossary_tooltip text="Score implementation (CLI)" term_id="score" >}} should report a syntax error.
+> If the referenced resource or its property aren't defined, the {{< glossary_tooltip text="Score implementation (CLI)" term_id="score" >}} should report a syntax error.
```

Comment:
This statement refers to properties that should be defined. And we do not declare properties anymore. Should remove any mention of the properties.